### PR TITLE
Created Home Page 

### PR DESCRIPTION
**Pull request by surbhi84**
This PR consists of a Home page that has below feats

- Plain html converted to react.
- Theme.
- Theme is implemented using a context.
- Addition of a favicon.

### CODE REVIEW

**Review by jeetbhalani15 — APPROVED** [review]
Code with folder & UI looks good

File: public/index.html [hunk 1]
```diff
@@ -6,10 +6,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
-      name="description"
-      content="Web site created using create-react-app"
+      name="Frontend Quiz app"
```
Comment (SJTGSHIVAM) [inline public/index.html:9]: nice usage of meta tags

File: src/components/footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                href="https://medium.com/@surbhikukreti"
+                target="blank"
+                className="footer-icon"
+              >
+                <i className="fab fa-medium-m"></i>
+              </a>
+            </li>
+          </ul>
+        </div>
+        {/* <!-- end of icon-links --> */}
```
Comment (SJTGSHIVAM) [inline src/components/footer/Footer.tsx:50]: These comments are making it easier to read

**Review by SJTGSHIVAM — APPROVED** [review]
Things look good.

**Pull request merged**